### PR TITLE
[MMI] Added Institution-background unit test

### PR DESCRIPTION
**Pull request by albertolive**
## Explanation

Added Institution-background unit test

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [x] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/store/institutional/institution-background.test.js [hunk 1]
```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
       expect(setWaitForConfirmDeepLinkDialog).toBeDefined();
     });
   });
+
+  describe('#showInteractiveReplacementTokenBanner', () => {
```
Comment (DDDDDanica) [inline ui/store/institutional/institution-background.test.js:127]: Can you add test for error handling as well :) 
```
if (err) {
        dispatch(displayWarning(err.message));
        throw new Error(err.message);
      }
```

File: ui/store/institutional/institution-background.test.js [hunk 2]
```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
...
+      })(dispatch);
+
+      expect(submitRequestToBackground).toHaveBeenCalledWith(
+        'showInteractiveReplacementTokenBanner',
+        [{ url: 'testUrl', oldRefreshToken: 'testToken' }],
+      );
+    });
+  });
+
+  describe('#setTypedMessageInProgress', () => {
```
Comment (DDDDDanica) [inline ui/store/institutional/institution-background.test.js:143]: QUESTION: In the code, 
```
export function setTypedMessageInProgress(msgId: string) {
  return async (dispatch: MetaMaskReduxDispatch) => {
    dispatch(showLoadingIndication());
    try {
      await submitRequestToBackground('setTypedMessageInProgress', [msgId]);
    } catch (error: any) {
      log.error(error);
      dispatch(displayWarning(error.message));
    } finally {
      await forceUpdateMetamaskState(dispatch);
      dispatch(hideLoadingIndication());
    }
  };
}
```

do we need to `await forceUpdateMetamaskState(dispatch);` if we have error? i feel like this should be removed under `await submitRequestToBackground('setTypedMessageInProgress', [msgId]);`

wdyt?

File: ui/store/institutional/institution-background.test.js [hunk 3]
```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
...
+      expect(submitRequestToBackground).toHaveBeenCalledWith(
+        'setTypedMessageInProgress',
+        ['testMsgId'],
+      );
+      expect(forceUpdateMetamaskState).toHaveBeenCalledWith(dispatch);
+      expect(hideLoadingIndication).toHaveBeenCalled();
+    });
+  });
+
+  describe('#setPersonalMessageInProgress', () => {
```
Comment (DDDDDanica) [inline ui/store/institutional/institution-background.test.js:159]: would be great to have error handling tests as well ;)

File: ui/store/institutional/institution-background.test.js (hunk 2 above)
Comment (zone-live) [reply ui/store/institutional/institution-background.test.js:143]: It's correct to have the `await forceUpdateMetamaskState(dispatch);` in the `finally` block 👍🏼

**Review by zone-live — APPROVED** [review]

File: ui/store/institutional/institution-background.test.js (hunk 3 above)
Comment (albertolive) [reply ui/store/institutional/institution-background.test.js:159]: Hey @DDDDDanica, at the moment we are not going to test the error parts

File: ui/store/institutional/institution-background.test.js (hunk 1 above)
Comment (albertolive) [reply ui/store/institutional/institution-background.test.js:127]: Hey @DDDDDanica, at the moment we are not going to test the error parts

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**